### ISSUE TITLE
ng github-pages:deploy deleted my project

### ISSUE THREAD

**Issue by YonatanKra** [angular/angular-cli#3030]
I've created a sample project, and then tried to upload it to github.
Instead of loading the project to github, it just deleted my local project completely - including the git commits, so there is no way back!
Any idea why this happened?
Here's the command I've used:
ng github-pages:deploy --message "First commit"

**Comment by johnazre** [comment]
Did it delete the folder, too? You can't access git within that directory? Did you push it to github before deploying it? Is it possible you changed branches and that's why there's nothing there?

**Comment by YonatanKra** [comment]
My bad - it was indeed a different branch. Didn't notice it checked out a branch.
Now all is wel on my sidel :)

**Comment by priyeshn** [comment]
Getting "Failed to create GitHub repo. Error: 401 Unauthorized" when tries to deploys angular 2 app to my organization guthub page  . Asking for a token and fails with 401

 **ng github-pages:deploy**  

? Please enter GitHub token you just created (used only once to create the repo): <token masked>
? and your GitHub user name: <userid>
Failed to create GitHub repo. Error: 401 Unauthorized

**Comment by Equinox** [comment]
@priyeshn You're commenting on a completely unrelated issue, and without full information also.

You should create a [new issue](https://github.com/angular/angular-cli/issues/new), ensuring to fill out the template provided and including all relevant information.

You didn't include in your above comment whether or not you created the GitHub token -- might be worth adding that in.
